symfony/plugins/orangehrmAdminPlugin/client/jobCategory/App.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <!-- TODO: Extend from base layout -->
+  <div class="container">
+   <job-category-list v-if="activeView==1" @on-add-item="changeView" @on-edit-item="changeView"></job-category-list>
+   <job-category-single v-if="activeView==2" @on-cancel="changeView" title="Add Job Category" action="Add" :jobCategory="null"></job-category-single>
+   <job-category-single v-if="activeView==3" @on-cancel="changeView" title="Edit Job Category" action="Update" :jobCategory="payload"></job-category-single>
```

Comment:
Can you format VUE files `prettier` standards

Reply:
Formatted with prettier.